gulpfile.js

```diff
@@ -0,0 +1,51 @@
+var path = require('path');
```

Comment:
OK, I think I know what you want to do with these tasks. Why not use `npm link`?